scripts/generate_docs.sc

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env amm
+
+/**
+  *
+  * DO NOT RUN THIS SCRIPT FROM THE SHELL!
+  *
+  * run ./doc_generation_launcher.sh instead
```

Comment:
this doesn't make sense

Reply:
for this change to run `./doc_generation_launcher.sh` 
add that preconditions for launching this script are in the bash file.

I would much prefer you fail this script if run without some pre-condition.